.github/workflows/create-gem-bump-branch.yml

```diff
@@ -0,0 +1,42 @@
+name: Create Gem Bump Branch
+on:
+  workflow_dispatch:
+    inputs:
+      new-gem-version:
+        type: choice
+        options:
+          - major
+          - minor
+          - patch
+          - pre
+          - release
+        required: true
+
+jobs:
+  release:
+    name: Bump Gem / Create PR
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - run: |
+          bundle exec gem bump --version ${{ github.event.inputs.new-gem-version || 'patch' }} --no-commit
```

Comment:
Do we need `|| 'patch` if it's a required input? Presumably we can just set a `default: 'patch` instead?

Reply:
Yeah that was used for testing with `on: push` - will remove.
Don't think defaults are passed through in that case